src/DssSpell.t.sol

```diff
@@ -2571,341 +2321,241 @@ contract DssSpellTest is DSTest, DSMath {
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_WBTC")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_ZRX")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_ZRX")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_KNC")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_KNC")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_MANA")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_MANA")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_USDT")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_USDT")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_COMP")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_COMP")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_LRC")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_LRC")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_LINK")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_LINK")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_BAL")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_BAL")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_YFI")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_YFI")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_UNI")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_UNI")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_AAVE")).src()).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(OsmAbstract(addr.addr("PIP_AAVE")).src()).orcl(LISKO), 1);
-        assertEq(MedianAbstract(ETHBTC).orcl(DEFI_SAVER), 0);
-        assertEq(MedianAbstract(ETHBTC).orcl(LISKO), 1);
+    address constant GOV_MULSTISIG  = 0x73f09254a81e1F835Ee442d1b3262c1f1d7A13ff;
+    address constant RISK_MULSTISIG = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
+    address constant RWF_MULSTISIG  = 0x9e1585d9CA64243CE43D42f7dD7333190F66Ca09;
+    address constant GRO_MULSTISIG  = 0x7800C137A645c07132886539217ce192b9F0528e;
+    address constant CP_MULSTISIG   = 0x6A0Ce7dBb43Fe537E3Fd0Be12dc1882393895237;
+
+    uint256 constant GOV_MONTHLY_EXPENSE  = 80_000;
+    uint256 constant RISK_MONTHLY_EXPENSE = 100_500;
+    uint256 constant RWF_MONTHLY_EXPENSE  = 40_000;
+    uint256 constant GRO_MONTHLY_EXPENSE  = 126_117;
+    uint256 constant CP_MONTHLY_EXPENSE   = 44_375;
```

Comment:
should this go to `specific for this spell` section?

Reply:
It could, but as we discussed, it seemed better to have them packed to the only test using them.